code/backtracking/src/sudoku_solve/sudoku_solveNxN.cpp

```diff
@@ -0,0 +1,96 @@
+
+#include<bits/stdc++.h>
+
+using namespace std;
```

Comment:
Don't use this.

Reply:
It's universal header file

Reply:
@aryangulati It's also bad practice. Use `std::` everywhere.

Reply:
https://isocpp.org/wiki/faq/coding-standards#using-namespace-std
either of the way is fine

Reply:
also most of the code use this due to decrease the complexity you can see many in the organisation ,too see this 
https://github.com/OpenGenus/cosmos/blob/master/code/backtracking/src/sudoku_solve/sudokusolve.cpp

Reply:
@ The established coding standard in this repository is to not use using namespace std. Even the link you put:

> Look, the whole point of namespaces is to prevent namespace collisions between two independently developed piles of code. The using-directive (that’s the technical name for using namespace XYZ) effectively dumps one namespace into another, which can subvert that goal. The using-directive exists for legacy C++ code and to ease the transition to namespaces, **but you probably shouldn’t use it on a regular basis, at least not in your new C++ code.**

So please remove the using directive. (More info here: https://stackoverflow.com/q/1452721/6525260). In the end it comes down to consistency and best standards.